linux_os/guide/services/sssd/sssd-ldap/sssd_ldap_start_tls/tests/ad_id_provider_and_tls_false.notapplicable.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_stig
+
+. $SHARED/setup_config_files.sh
+setup_correct_sssd_config
```

Comment:
I think it should also ensure if sssd is installed.

Reply:
ok, I will take a look, from others test scenarios this should be enough:

```
yum -y install /usr/lib/systemd/system/sssd.service
systemctl enable sssd
```

Reply:
That's amazing! It helps people like me who use the minimal RHEL VM for testing, as sssd isn't by default.

> yum -y install /usr/lib/systemd/system/sssd.service
> systemctl enable sssd

^ Probably vice-versa :smile:

Reply:
What do you mean by vice-versa here? Do you mean this: 03fe81459f0628bb08169086285cdf1cfef64e5b ?